### PR TITLE
login: remove old user_sess cookie support

### PR DESCRIPTION
**Pull request by bonifaido**
Signed-off-by: Nandor Kracser <bonifaido@gmail.com>

| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| API breaks?     | no
| Deprecations?   | yes
| Related tickets | N/A
| License         | Apache 2.0

### What's in this PR?

Removing the presence check fo the user_sess cookie, which is not sent by Pipeline a long ago.

### Checklist

- [x] Code meets the [Developer Guide](https://github.com/banzaicloud/developer-guide)

### CODE REVIEW

**Review by sagikazarmark — APPROVED** [review]

**Review by orymate — APPROVED** [review]

File: .licensei.toml [hunk 1]
```diff
@@ -20,6 +20,7 @@ ignored = [
     # Vanity URLS :\
     "cloud.google.com/go",
     "emperror.dev/errors",
+    "gopkg.in/square/go-jose.v2",
```
Comment (pregnor) [inline .licensei.toml:23]: **Note**: I think this should be part of a different PR.

Comment (bonifaido) [reply .licensei.toml:23]: I haven't changed anything regarding JWT libs in this PR, but the build is red without this line, probably some indirect dependency changed in the background: https://app.circleci.com/pipelines/github/banzaicloud/banzai-cli/1370/workflows/fba26aae-bfba-4974-86b5-301da0a90381/jobs/2478

Comment (pregnor) [reply .licensei.toml:23]: Interesting, because the corresponding dependency was added more than a year ago. Has the packaged changed its license?

I'm fine merging it this way then (though let's make sure we don't overlook a possibly problematic license in case it did change).

(A separate fix PR and this user_sess removal PR would be preferred to avoid possibly reverting the license check fix in case we need to revert the user_sess removal code, but I don't want to split hairs over this, just keep in mind for the future.)
